src/plugins/console/server/routes/api/console/autocomplete_entities/register_get_route.ts

```diff
@@ -71,11 +90,36 @@ export function registerGetRoute({ router, lib: { handleEsError } }: RouteDepend
         }
 
         const esClient = (await ctx.core).elasticsearch.client;
-        const mappings = await getMappings(esClient, settings);
-        const aliases = await getAliases(esClient, settings);
-        const dataStreams = await getDataStreams(esClient, settings);
-        const [legacyTemplates = {}, indexTemplates = {}, componentTemplates = {}] =
-          await getTemplates(esClient, settings);
+
+        // Wait for all requests to complete, in case one of them fails return the successfull ones
+        const results = await Promise.allSettled([
+          getMappings(esClient, settings),
+          getAliases(esClient, settings),
+          getDataStreams(esClient, settings),
+          getTemplates(esClient, settings),
+        ]);
+
+        const map = results.map((result, idx) => {
+          // If the request was successful, return the result
+          if (result.status === 'fulfilled') {
+            return result.value;
+          }
+
+          // getTemplates returns an array of results, so we need to handle it in case of failure
+          if (idx === results.length - 1) {
```

Comment:
nit: this map looks is not obvious. Maybe we can move that extra logic inside `getTemplates` promise?

Reply:
addressed with 0f7f5d8a3986ab32af225c2bf47cb03b6326b646